packages/block-library/src/more/edit.native.js

```diff
@@ -2,85 +2,47 @@
  * External dependencies
  */
 import { View } from 'react-native';
+import Hr from 'react-native-hr';
 
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
-import {
-	getDefaultBlockName,
-	createBlock,
-} from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
-import { PlainText } from '@wordpress/block-editor';
 import styles from './editor.scss';
 
 export default class MoreEdit extends Component {
 	constructor() {
 		super( ...arguments );
-		this.onChangeInput = this.onChangeInput.bind( this );
 
 		this.state = {
 			defaultText: __( 'Read more' ),
 		};
 	}
 
-	onChangeInput( newValue ) {
-		// Detect Enter.key and add new empty block after.
-		// Note: This is detected after the fact, and the newline could be visible on the block
-		// for a very small time. This is OK for the alpha, we will revisit the logic later.
-		// See https://github.com/wordpress-mobile/gutenberg-mobile/issues/324
-		if ( newValue.indexOf( '\n' ) !== -1 ) {
-			const { insertBlocksAfter } = this.props;
-			insertBlocksAfter( [ createBlock( getDefaultBlockName() ) ] );
-			return;
-		}
-		// Set defaultText to an empty string, allowing the user to clear/replace the input field's text
-		this.setState( {
-			defaultText: '',
-		} );
-		const value = newValue.length === 0 ? undefined : newValue;
-		this.props.setAttributes( { customText: value } );
-	}
-
-	renderLine() {
-		return <View style={ styles[ 'block-library-more__line' ] } />;
-	}
-
-	renderText() {
-		const { attributes, onFocus, onBlur } = this.props;
-		const { customText } = attributes;
+	render() {
+		const { customText } = this.props.attributes;
 		const { defaultText } = this.state;
-		const value = customText !== undefined ? customText : defaultText;
+		const content = customText || defaultText;
 
 		return (
-			<View>
-				<PlainText
-					style={ styles[ 'block-library-more__text' ] }
-					value={ value }
-					multiline={ true }
-					underlineColorAndroid="transparent"
-					onChange={ this.onChangeInput }
-					placeholder={ defaultText }
-					isSelected={ this.props.isSelected }
-					onFocus={ onFocus }
-					onBlur={ onBlur }
+			<View
+				accessible={ ! this.props.isSelected }
+				accessibilityLabel={ __( 'More block' ) }
```

Comment:
Should this be "Read more block"? More feels a bit ambiguous

Reply:
Modified the accessibility config based on https://github.com/wordpress-mobile/gutenberg-mobile/pull/937 

Now all blocks will have a default label based on the gutenberg-defined block name.